src/components/cc-toggle/cc-toggle.js

```diff
@@ -172,13 +172,12 @@ export class CcToggle extends LitElement {
           --cc-toggle-img-filter-selected: none;
           --height: 2em;
 
-          display: inline-flex;
+          display: block;
```

Comment:
**suggestion:** I didn't realize it when we discussed in sync but most form elements have a default display that is `inline-block`.

Maybe we should respect that ? Also, changing the `display` property of an element is a breaking change so we may need to rethink that and make sure there is no other solution.

I wonder if you face the same issue with `cc-input-text` and `inline` mode?

Reply:
I'll test a few things and I'll get back to you as soon as possible